backend-api/infra/monitoring/alarms.yaml

```
@@ -1889,3 +1889,46 @@ Resources:
           Label: "Number of 5XX errors returned as a percentage"
           ReturnData: false
           Expression: (errorSum5XX/invocations) * 100
+
+  AsyncIssueBiometricCredentialInvalidSqsEventAlarm:
+    Type: AWS::CloudWatch::Alarm
+    DependsOn:
+      - "AsyncIssueBiometricCredentialInvalidSqsEventMetricFilter"
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:platform-alarms-sns-warning"
+      OKActions:
+        - !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:platform-alarms-sns-warning"
+      InsufficientDataActions: []
+      AlarmDescription: "Fires when a log event with messageCode MOBILE_ASYNC_ISSUE_BIOMETRIC_CREDENTIAL_INVALID_SQS_EVENT is detected."
+      AlarmName: !Sub "${AWS::StackName}-issue-biometric-credential-lambda-invalid-sqs-event"
+      Namespace: !Sub "${AWS::StackName}/LogMessages"
+      MetricName: AsyncIssueBiometricCredentialMessageCode
+      Dimensions:
+          - Name: MessageCode
+            Value: MOBILE_ASYNC_ISSUE_BIOMETRIC_CREDENTIAL_INVALID_SQS_EVENT
+          - Name: Version
+            Value: $LATEST
```

When I update this (see below), the alarm stops firing

```suggestion
            Value: !GetAtt AsyncIssueBiometricCredentialFunction.Version.Version
```